src/app/common/model/confirmation-modal-data.ts

```diff
@@ -0,0 +1,6 @@
+export interface IConfirmationModalData{
```

Comment:
Also you should have related files / directories in the same directory. So for all of the confirmation modal files I would propose such directory structure:
```typescript
// common/confirmation-modal/confirmation-modal.component.ts
// common/confirmation-modal/confirmation-modal.component.html
// common/confirmation-modal/confirmation-modal.component.scss
// common/confirmation-modal/confirmation-modal.service.ts
// common/confirmation-modal/confirmation-modal.types.ts
// common/confirmation-modal/spec/confirmation-modal.component.spec.ts
// common/confirmation-modal/spec/confirmation-modal.service.spec.ts
```